### PR TITLE
[M] Added the new Candlepin Authority R2 Certificate (ENT-4853)

### PR DESCRIPTION
**Pull request by JoshAlbrecht23**
**Validation Steps:**
In a vagrant VM I added the candlepin-redhat-ca.crt file and the new candlepin-redhat-r2-ca.crt file to the /etc/candlepin/certs/upstream/ directory to be used for importing the manifests.

Importing the Hosted Stage manifest that uses the new R2 certificate:
import_stage_manifest

Importing the Hosted Prod manifest that uses the current certificate:
import_prod_manifest

### CODE REVIEW

File: config/candlepin/candlepin-redhat-r2-ca.crt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ExsrfgyGdkk67RtcMs8T9GimDxd0xiUXsbVkE6ATeSK9ELha01jUkGT/y9WDrTpr
+U01dTsrDLws9KmWbxWrU4OHswRe2NznbGRKgTs83efUkolAYlMCuT8jx6P5kNLWm
+NE383TrUtFrllA5/JO8QviWrmvVLR+rOBjE1PHswhsvsKRaxHkcB7XLHDexYcIAs
+hvoe7YAgqmFEcroLvpmeUtg5fOMlVb4ORIsMf16vqmXNrSGywJLw3tUUEYWuEChh
+WshcbRf5EE1ROao8zGYnLhVF4OIH4ZMy3dFEpbXaLdLleWnOX6BJEvYL/71Xsb1l
+WCSgufrk+or9Lz+SMJmJBzHqMqdGIFMINHL+Ritv63uhPMTklZec7ePEbtU1pbtI
+NK74t9l/7gIGz05fLRJGDAVEmC6LFRpyyZSh6qELwwNVot1y0xojPSy3dbLmUxuO
+DwBHkHsxstyyg2py9rPYj3Ik8rXpRp5yRW1+ehFI8PenMOA7RWT+u/xor6XrpiP9
+3N4ml0cLV0ASeWU=
+-----END CERTIFICATE-----
```
Comment (Ceiu) [inline config/candlepin/candlepin-redhat-r2-ca.crt:40]: End the file with a new line

**Review by Ceiu — CHANGES_REQUESTED** [review]
Just need that missing new line and this should be good.

For future reference, images in the PRs are harder to digest than raw text. Use the block formatting or code formatting syntax for showing such output or commands for validation.

File: config/candlepin/candlepin-redhat-r2-ca.crt (hunk 1 above)
Comment (JoshAlbrecht23) [reply config/candlepin/candlepin-redhat-r2-ca.crt]: Added a new line to the end of the file

**Comment by JoshAlbrecht23** [comment]
That's a good point and that makes more sense than using an image! I'll use the raw text with the block or code formatting for displaying output in future PRs.

**Review by Ceiu — APPROVED** [review]

**Pull request merged**